clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -771,7 +771,7 @@ private void processDisconnection(List<ClientResponse> responses,
                     nodeId, disconnectState.remoteAddress());
                 break;
             case NOT_CONNECTED:
-                log.warn("Connection to node {} ({}) could not be established. Broker may not be available.", nodeId, disconnectState.remoteAddress());
+                log.debug("Connection to node {} ({}) could not be established. Broker may not be available.", nodeId, disconnectState.remoteAddress());
```

Comment:
what's the rationale for this change?

Reply:
I'll revert it. It was just really spammy when I was testing failover.